src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js

```diff
@@ -639,75 +629,73 @@ window.CELEMENTS.presentation.SlideShow = function(containerId) {
        * prerequisite: slideWrapper MUST have a height and width asigned (e.g. by resize)
        * @TODO move to CelementsSlides js class.
        */
-      _centerCurrentSlide : function(slideWrapperIn) {
-        var _me = this;
-        var slideWrapper = slideWrapperIn || _me._getSlideWrapper();
-        var slideRoot = _me._getSlideRootElem(slideWrapper);
-//        console.log('_centerCurrentSlide for ', slideWrapper, slideRoot);
-//        slideWrapper.setStyle({
-//          'position' : 'absolute',
-//          'width' : 'auto',
-//          'height' : 'auto',
-//          'marginLeft' : 0,
-//          'marginRight' : 0
-//        });
-//        slideRoot.setStyle({
-//          'position' : 'relative',
-//          'top' : 0
-//        });
+      _centerCurrentSlide: function (slideWrapperIn) {
+        const _me = this;
+        const slideWrapper = slideWrapperIn || _me._getSlideWrapper();
+        const slideRoot = _me._getSlideRootElem(slideWrapper);
         //we cannot read element dimension if any parent is hidden (display:none)
-        var hiddenParentElems = [];
-        slideWrapper.ancestors().each(function(parentElem) {
+        const hiddenParentElems = [];
+        slideWrapper.ancestors().each(function (parentElem) {
           if (!parentElem.visible()) {
             hiddenParentElems.push(parentElem);
             parentElem.show();
           }
         });
-        //use jquery to get dimensions, because it works correctly inside iframes.
-        var slideOuterHeight = $j(slideRoot).height();
-        var slideOuterWidth = $j(slideRoot).width();
-        var parentDiv = _me._htmlContainer;
-        var parentHeight = parentDiv.getHeight();
-        var parentWidth = parentDiv.getWidth();
-//        console.log('_centerCurrentSlide for dim ', slideOuterWidth, slideOuterHeight,
-//            parentWidth, parentHeight);
+        //ensure that the width is not influenced by the parents by setting position to absolute
+        slideRoot.setStyle({
+          'position': 'absolute'
+        });
+        //use scrollWidth and scrollHeight, because it works correctly inside iframes.
+        const slideOuterHeight = slideRoot.scrollHeight;
+        const slideOuterWidth = slideRoot.scrollWidth;
+        if (_me._debug) {
+          console.log('_centerCurrentSlide slideRoot dim: ', _me._htmlContainerId, slideOuterWidth,
```

Comment:
```suggestion
          console.debug('_centerCurrentSlide slideRoot dim: ', _me._htmlContainerId, slideOuterWidth,
```
and all following occurences of `console.log` within debug check